plugins/woocommerce/includes/admin/importers/views/html-product-csv-import-form.php

```diff
@@ -78,6 +78,20 @@
 					<th><label><?php esc_html_e( 'Use previous column mapping preferences?', 'woocommerce' ); ?></label><br/></th>
 					<td><input type="checkbox" id="woocommerce-importer-map-preferences" name="map_preferences" value="1" /></td>
 				</tr>
+				<tr class="woocommerce-importer-advanced hidden">
+					<th><label><?php esc_html_e( 'Character encoding of the file', 'woocommerce' ); ?></label><br/></th>
+					<td><select id="woocommerce-importer-character-encoding" name="character_encoding">
+							<option value="" selected><?php esc_html_e( 'Autodetect', 'woocommerce' ); ?></option>
+							<?php
+							$encodings = mb_list_encodings();
+							sort( $encodings, SORT_STRING );
```

Comment:
Non-essential change we could consider:

```suggestion
							sort( $encodings, SORT_NATURAL );
```

This would provide us with the following (which I think most users would find slightly easier to navigate):

```
     "ISO-8859-1",
     "ISO-8859-2",
     "ISO-8859-3",
     "ISO-8859-4",
     "ISO-8859-5",
     "ISO-8859-6",
     "ISO-8859-7",
     "ISO-8859-8",
     "ISO-8859-9",
     "ISO-8859-10",
     "ISO-8859-13",
     "ISO-8859-14",
     "ISO-8859-15",
     "ISO-8859-16",
```

Instead of:

```
     "ISO-8859-1",
     "ISO-8859-10",
     "ISO-8859-13",
     "ISO-8859-14",
     "ISO-8859-15",
     "ISO-8859-16",
     "ISO-8859-2",
     "ISO-8859-3",
     "ISO-8859-4",
     "ISO-8859-5",
     "ISO-8859-6",
     "ISO-8859-7",
     "ISO-8859-8",
     "ISO-8859-9",
```